File: server_nodejs/src/models/cart_item.js
// Cart_ item (clientID, itemID, quantity)
module.exports = (sequelize, Sequelize) => {
    class CartItem extends Sequelize.Model {}

    CartItem.init(
        {
            clientid: {
                type: Sequelize.UUID,
                primaryKey: true,
            },
            itemid: {
                type: Sequelize.UUID,
                primaryKey: true,
            },
            quantity: {
                type: Sequelize.INTEGER,
                allowNull: false,
            },
        },
        {
            sequelize,
            modelName: 'cart_item',
            timestamps: true,
            paranoid: false,
            underscored: true,
        },
    );

    CartItem.associate = (models) => {
        CartItem.belongsTo(models.item, {
            foreignKey: 'itemid',
        });
        CartItem.belongsTo(models.client_account, {
            foreignKey: 'clientid',
        });
    };

    return CartItem;
};

File: server_asp/src/models/CartItem.cs
using System;
using System.Collections.Generic;

namespace shieldtify.models;

public partial class CartItem
{
    public Guid Clientid { get; set; }

    public Guid Itemid { get; set; }

    public int Quantity { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }


    public virtual ClientAccount Client { get; set; } = null!;

    public virtual Item Item { get; set; } = null!;
}
